packages/logger/src/Logger.ts

```diff
@@ -18,6 +18,30 @@ import type {
   PowertoolLogData,
 } from './types';
 
+/**
+ *
+ * # Logger class
+ *
+ * ## Intro
+ *
+ * The Logger utility provides an opinionated logger with output structured as JSON.
+ *
+ * ## Key features
+ *
+ * - Capture key fields from Lambda context, cold start and structures logging output as JSON
+ * - Log Lambda context when instructed (disabled by default)
+ * - Log sampling prints all logs for a percentage of invocations (disabled by default)
+ * - Append additional keys to structured log at any point in time
+ *
+ * ## Basic usage
+ *
+ * import { Logger } from "@aws-lambda-powertools/logger";
```

Comment:
I'd wrap these examples in a code block so they are rendered like this:
![image](https://user-images.githubusercontent.com/7353869/150307402-c50219ad-9f8a-4b64-b78e-7fc99a76141e.png)

See here: https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/packages/metrics/src/Metrics.ts#L40-L62